libs/structs/structvar/structvar_test.go

```diff
@@ -17,20 +17,24 @@ type TestObj struct {
 
 // newTestStructVar creates a fresh StructVar instance for testing
 func newTestStructVar() *structvar.StructVar {
-	return &structvar.StructVar{
-		Value: &TestObj{
+	sv, err := structvar.NewStructVar(
+		&TestObj{
 			Name: "OldName",
 			Age:  25,
 			Tags: map[string]string{
 				"env": "old_env",
 			},
 		},
-		Refs: map[string]string{
+		map[string]string{
 			"name":            "${var.name}",
 			"age":             "${var.age}",
 			"tags['version']": "${var.version}",
 		},
+	)
+	if err != nil {
+		panic(err)
```

Comment:
require no error instead of panic? Panic can cause other tests in the test suite to bail out.